ui/admin/package.json

```diff
@@ -102,7 +102,7 @@
     "sinon": "^14.0.0",
     "stylelint": "^15.10.1",
     "stylelint-config-prettier-scss": "^0.0.1",
-    "stylelint-config-standard-scss": "^6.0.0",
```

Comment:
I will suggest to not perform `ui/admin` upgrades on a PR that is expected to perform `ui/desktop` unless there is a reason for it, can you ellaborate why we have `admin` changes within this PR?

Reply:
@calcaide will do